src/sentry/integrations/msteams/webhook.py

```diff
@@ -200,6 +202,55 @@ def handle_member_removed(self, request):
         integration.delete()
         return self.respond(status=204)
 
+    def make_action_data(self, identity, data):
```

Comment:
@KKaempen I thin the signature for this function should be:
```
def make_action_data(self, data, user_id):
```
Data should go first because it's the more important field and we don't need the full identity object, just the user_id.